nix/docker.nix

```diff
@@ -136,9 +136,13 @@ let
         set -euo pipefail
         ${scripts.mainnet.db-sync.passthru.service.restoreSnapshotScript}
 
-        exec $DBSYNC \
-          --schema-dir ${../schema} \
-          --state-dir ${scripts.mainnet.db-sync.passthru.service.stateDir} $@
+        if [[ -n "$DISABLE_LEDGER" ]]; then
```

Comment:
I understand you just want to check whether `DISABLE_LEDGER` is set or not but, given you write `DISABLE_LEDGER=true` in the documentation above, I am worried someone will do `DISABLE_LEDGER=false` :joy:.
Maybe checking for an explicit value of `DISABLE_LEDGER` (e.g. "true" or 1) could be a safer option?

Reply:
That's a good  point. After looking around it, it looks like there's an existing pattern (`RECREATE_DB=Y`), so I think it would make sense to follow that pattern. Source: https://github.com/input-output-hk/cardano-db-sync/blob/master/scripts/postgresql-setup.sh#L311